### PR TITLE
minor fixes

### PR DESCRIPTION
**Pull request by jaredkz**
Linting, set country to required.

### CODE REVIEW

**Comment by jaredkz** [comment]
Closes #434 
Closes #424 
Closes #416 
Closes #413 
Closes #407

**Comment by jaredkz** [comment]
Closes #412 
Closes #420 
Readme was recently updated.

Closes #402 
Missing currency MVR added.

**Comment by jaredkz** [comment]
Credits: 
[Fjuro](https://github.com/Fjuro) for Czech language support.
[itsmelaxman:feat-translation-np](https://github.com/itsmelaxman/Paisa/tree/feat-translation-np) for Nepalese language support.
[i-like-trains-de:patch-1](https://github.com/i-like-trains-de/Paisa/tree/patch-1) for German language updates.
[pioucraft:patch-1](https://github.com/pioucraft/Paisa/tree/patch-1) for french language updates.
[AlexHladin:add-ukrainian-translation](https://github.com/AlexHladin/Paisa/tree/add-ukrainian-translation) for Ukranian language support.
[luisdeeev](https://github.com/luisdeeev/Paisa/commits?author=luisdeeev) for Portuguese (BR) language support.

**Comment by h4h13** [comment]
Hey can you explain what all has been done on this PR

File: lib/core/constants/constants.dart
Comment (h4h13) [inline lib/core/constants/constants.dart:1]: Why did you change package name

Comment (jaredkz) [reply lib/core/constants/constants.dart:1]: I was playing with github actions and forgot to change it back.

**Comment by jaredkz** [comment]
Considering the multiple recent changes, I'll rebase to the latest version and explain any potential changes as many of the issues have been resolved.

**Comment by h4h13** [comment]
Cool thanks
